pipeline-model-definition/src/main/groovy/org/jenkinsci/plugins/pipeline/modeldefinition/parser/JSONParser.groovy

```diff
@@ -103,8 +103,8 @@ class JSONParser {
                 case 'tools':
                     pipelineDef.tools = parseTools(pipelineJson.getJSONArray("tools"))
                     break
-                case 'properties':
-                    pipelineDef.properties = parseJobProperties(pipelineJson.getJSONObject("properties"))
+                case 'options':
```

Comment:
Shoudl perhaps leave an information warning if someone still tries to use `properties` or has it not been that long since the rename? :)